### PR TITLE
initial content for vol group feature

### PR DESCRIPTION
**Pull request by rivkap**

### CODE REVIEW

File: docs/content/configuration/creating_volumestorageclass.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Use the `SpaceEfficiency` parameters for each storage system, as defined in the
       parameters:
         pool: demo-pool
         io_group: demo-iogrp             # Optional.
+        volume_group: demo-volumegroup             # Optional.
```
Comment (oriyarde) [inline docs/content/configuration/creating_volumestorageclass.md:59]:
```suggestion
        volume_group: demo-volumegroup  # Optional.
```
(should be the same as in [here](https://github.com/IBM/ibm-block-csi-driver/blob/0de102723dd808cf55e95ce92035cc6b89e9f5d1/deploy/kubernetes/examples/demo-storageclass.yaml#L9))

File: docs/content/configuration/creating_storageclass_topology_aware.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The various `by_management_id` parameters are chosen within the following hierar
           # non-csi.storage.k8s.io parameters may be specified in by_management_id per system and/or outside by_management_id as the cross-system default.
 
           by_management_id: '{"demo-management-id-1":{"pool":"demo-pool-1","SpaceEfficiency":"dedup_compressed","volume_name_prefix":"demo-prefix-1"},
-                              "demo-management-id-2":{"pool":"demo-pool-2","volume_name_prefix":"demo-prefix-2", "io_group": "demo-iogrp"}}'  # Optional.
+                              "demo-management-id-2":{"pool":"demo-pool-2","volume_name_prefix":"demo-prefix-2", "io_group": "demo-iogrp": "volume_group": "demo-volumegroup"}}'  # Optional.
```
[the remark this answers is not in the document]
Comment (rivkap) [reply docs/content/configuration/creating_storageclass_topology_aware.md:27]: so just in the regular storage class yaml?

**Review by oriyarde — APPROVED** [review]
LGTM

File: docs/content/configuration/creating_storageclass_topology_aware.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oriyarde) [reply docs/content/configuration/creating_storageclass_topology_aware.md:27]: yes

[the remark this answers is not in the document]
Comment (rivkap) [reply docs/content/configuration/creating_storageclass_topology_aware.md:27]: changed back

File: docs/content/configuration/creating_volumestorageclass.md (hunk 1 above)
Comment (rivkap) [reply docs/content/configuration/creating_volumestorageclass.md]: updated spacing. it just got copied wrong

**Pull request merged**